src/components/navbar.jsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import './bundle.css';
+
+const awsTopContributorsURL = '';
+const awsTiersURL = 'http://3.15.215.14:3101/';
+const awsDescriptionURL = 'http://54.193.6.166:3663/';
```

Comment:
For the future, there is probably another way to do this without using global vars. Maybe a file/module that holds all of your urls that gets imported or required? An additional benefit would be that if you needed them in any other place they would be easy to grab. It would also make your URLS east to update/change/find for updating.

Reply:
Yeah, I was thinking about putting it into a separate file, perhaps with my helper functions or in a separate routing file, but didn't feel like it was necessary on this one since each is only being called once. This way they are at least lumped together for ease of location.